include/boost/geometry/strategies/cartesian/point_in_poly_winding.hpp

```diff
@@ -229,6 +217,29 @@ class cartesian_winding
     }
 };
 
+} // namespace detail
+#endif // DOXYGEN_NO_DETAIL
+
+/*!
+\brief Within detection using winding rule in cartesian coordinate system.
+\ingroup strategies
+\tparam Point_ \tparam_point
+\tparam PointOfSegment_ \tparam_segment_point
+\tparam CalculationType \tparam_calculation
+
+\qbk{
+[heading See also]
+[link geometry.reference.algorithms.within.within_3_with_strategy within (with strategy)]
+}
+ */
+template
+<
+    typename Point_ = void, // for backward compatibility
+    typename PointOfSegment_ = Point_, // for backward compatibility
+    typename CalculationType = void
+>
+class cartesian_winding
+  : public detail::cartesian_winding_base<typename side::services::default_strategy<cartesian_tag, CalculationType>::type, CalculationType> {};
```

Comment:
isn't it too long? If not skip my comment.

Reply:
It is indeed. Will fix.